shared/patches/react-native-reanimated+3.5.4.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/node_modules/react-native-reanimated/src/reanimated2/PlatformChecker.ts b/node_modules/react-native-reanimated/src/reanimated2/PlatformChecker.ts
+index 9b3fcb1..327cf21 100644
+--- a/node_modules/react-native-reanimated/src/reanimated2/PlatformChecker.ts
++++ b/node_modules/react-native-reanimated/src/reanimated2/PlatformChecker.ts
+@@ -44,6 +44,7 @@ export function isWindowAvailable() {
+ }
+ 
+ export function isReducedMotion() {
++return false;
```

Comment:
bottom sheet has issues with this on currently, the modal auto dismisses https://github.com/gorhom/react-native-bottom-sheet/issues/1560